server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1378,8 +1385,10 @@ boolean canSuspendTablets() {
       } catch (KeeperException | InterruptedException e) {
         throw new IllegalStateException("Exception setting up delegation-token key manager", e);
       }
-      authenticationTokenKeyManagerThread =
-          Threads.createThread("Delegation Token Key Manager", authenticationTokenKeyManager);
+      // TODO KEVIN RATHBUN managing delegation tokens seems like a critical function of the
+      // manager and this is not recreated on failures.
+      authenticationTokenKeyManagerThread = Threads
+          .createCriticalThread("Delegation Token Key Manager", authenticationTokenKeyManager);
```

Comment:
managing delegation tokens seems like a critical function of the manager and this is not recreated on failures.